prototype/p2p/messenger_test.go

```diff
@@ -0,0 +1,78 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"testing"
+)
+
+func TestMessager_Protobuff(t *testing.T) {
+	nonce, src, dst, level := int32(1), "10.0.0.1:1234", "9.0.0.2:5432", int32(2)
+
+	// test message instantiation
+	{
+		p2pmsg := Message(nonce, level, types.PocketTopic_P2P_PING, src, dst)
+
+		if p2pmsg.Topic != types.PocketTopic_P2P_PING {
```

Comment:
Highly recommend importing these dependencies:

```
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
```

The test could then be replaced with one line:

`assert.Equal(p2pmsg.Topic, types.PocketTopic_P2P_PING types.PocketTopic_P2P_PING, fmt.Sprintf("Protobuff messenger error: Failed to instantiate a ping message, expected topic: %s, got: %s", types.PocketTopic_P2P_PING, p2pmsg.Topic))`

See example here: https://github.com/pokt-network/consensus-pre-prototype/blob/main/pkg/consensus/leader_election/election_test.go

Reply:
Good suggestion